src/common/combocmn.cpp

```diff
@@ -1019,8 +1021,9 @@ void wxComboCtrlBase::CalculateAreas( int btnWidth )
     //   button width is set to default and blank button bg is not drawn
     if ( m_bmpNormal.IsOk() )
     {
-        int bmpReqWidth = m_bmpNormal.GetWidth();
-        int bmpReqHeight = m_bmpNormal.GetHeight();
+        wxBitmap currentBmp = m_bmpNormal.GetBitmapFor(this);
```

Comment:
This is minor, but I don't think we actually need the bitmap itself here, just its size, so it looks like calling `GetPreferredLogicalSizeFor()` should be enough, shouldn't it?